frontend/src/components/Cards/CardReport.tsx

```diff
@@ -0,0 +1,46 @@
+import { Card, CardDescription, CardFooter, CardTitle } from '../Card'
+import { Button } from '../Button'
```

Comment:
Nói rồi mà, không import tay:
```suggestion
import { Button } from '~/components/Button'
```